packages/@lwc/integration-karma/package.json

```diff
@@ -28,7 +29,7 @@
         "karma-chrome-launcher": "^3.2.0",
         "karma-coverage": "^2.2.1",
         "karma-jasmine": "^4.0.2",
-        "karma-sauce-launcher": "^4.3.6",
+        "karma-sauce-launcher": "nolanlawson/karma-sauce-launcher#e171723",
```

Comment:
I considered publishing to npm, but I guess I have to pay for `@nolanlawson/karma-sauce-launcher`. 😛 I like using the GitHub URL here because at least it's clear we own this fork.